calcom/cal.com
# Duplicate meeting within non-traditional calendars

## What does this PR do?

When user does not have any calendar but there is a non-traditional calendar app installed such as Close.com or HubSpot, the bookings were going through those apps twice.

<kbd><img width="175" alt="image" src="https://user-images.githubusercontent.com/467258/197796539-bc5d14cb-e2f7-4f77-aa46-51d05fd631e8.png"></kbd>

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?

Have no calendar installed and install a CRM. Book a meeting and see the meeting created just once in the CRM while also seeing the confirmation email with just a single app status for it.
